index.html

```diff
@@ -50,6 +50,7 @@ <h3>
 <li><a href="http://spotbugs.readthedocs.io/en/latest/maven.html">Maven</a></li>
 <li><a href="http://spotbugs.readthedocs.io/en/latest/gradle.html">Gradle</a></li>
 <li><a href="http://spotbugs.readthedocs.io/en/latest/eclipse.html">Eclipse</a></li>
+<li><a href="https://vimhelp.org/quickfix.txt.html#compiler-spotbugs">(Neo)Vim</a></li>
```

Comment:
Yes, it keeps being quite an Odyssey. It will eventually get there. Thank you for your consideration and feel free to contribute with your expertise, for exampling regarding the last question about the need for the setting of `-auxiliaryPath`.

Reply:
Dear @JuditKnoll , it finally went live